test/integration2/query/insert/inserts.spec.js

```diff
@@ -126,7 +126,7 @@ describe('Inserts', function () {
                 1,
                 TEST_TIMESTAMP,
               ],
-              [1]
```

Comment:
We'll either want this to be opt-in, or need to classify this as a breaking change rather than a fix, requiring a major version bump. 

Since it's only available in newer sqlite versions, I think it might be better handled as a config option where we can opt-in to this behavior for now, and then we can make this a breaking change later. 

I'll see what @kibertoad thinks though since I've been away from the project for quite a bit.

Reply:
It would be great to make this consistent across all dialects, I think MySQL is also doing it the other way around now.
Config option would require support across all dialects; I prefer a simpler solution, and don't mind going for 3.0.0; not like we are running out of numbers any time soon.